crankshaft-monitor/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+//! Crate for monitoring crankshaft events.
+use std::net::SocketAddr;
+
+use anyhow;
+use proto::{Event, monitor_server::MonitorServer};
+use server::MonitorService;
+use tokio::sync::broadcast;
+
+pub mod proto;
+pub mod server;
+
+const DEFAULT_CHANNEL_CAPACITY: usize = 16;
+
+/// The main external API to start the Crankshaft monitor.
+pub async fn start_monitoring(
```

Comment:
It appears that this function doesn't need to be `async`.

```suggestion
pub fn start_monitoring(
```